core-bundle/src/String/SimpleTokenExpressionLanguage.php

```diff
@@ -17,9 +17,9 @@
 
 class SimpleTokenExpressionLanguage extends ExpressionLanguage
 {
-    public function __construct(CacheItemPoolInterface|null $cache = null, \IteratorAggregate|null $taggedProviders = null)
+    public function __construct(CacheItemPoolInterface|null $cache = null, \Traversable|array|null $taggedProviders = null)
```

Comment:
```suggestion
    public function __construct(CacheItemPoolInterface|null $cache = null, iterable|null $taggedProviders = null)
```

Reply:
This won't work, as described in the checking code block

> when used in a union type, iterable is an alias for Traversable|array.
> https://www.php.net/manual/en/reflectionuniontype.gettypes.php#128871